plugins/compress/compress.cc

```diff
@@ -63,6 +62,8 @@ const int BROTLI_COMPRESSION_LEVEL = 6;
 const int BROTLI_LGW               = 16;
 #endif
 
+static const char *global_hidden_header_name = nullptr;
+
```

Comment:
I assume this is strictly speaking a correctness change rather than just a leak change.